.travis.yml

```diff
@@ -7,6 +7,7 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
```

Comment:
Please rebase the PR.

Reply:
fixed: https://github.com/utolosa002/php-gitlab-api/commit/068e0f43ce3fadb2778e28dd3bc7f5b51e5a2df1